### PR TITLE
Overhaul README

### PR DESCRIPTION
**Pull request by flyingzumwalt**
Radically simplifies the pm README, removing outdated info about the old PM process. This will reduce confusion as we settle on new PM process from the discussion & proposal in ipfs/pm#125 and ipfs/pm#131

Related to ipfs/pm#134

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,128 +7,33 @@
 
 > Project Management
 
-This is a repository for general ipfs project management.
+This is a repository for general IPFS project management.
 
-#### Please don't open issues here unless you know what you're doing.
+tl:dr; We have a **weekly, all-hands call**. The time and date is announced in [the issues](). Propose agenda items in the etherpad linked in that issue!
```
Comment (jbenet) [inline README.md]: - (not sure of typo or intentionally left blank) fill link to https://github.com/ipfs/pm/issues

Comment (flyingzumwalt) [inline README.md]: @RichardLitt we can delete this line, right?

**Comment by jbenet** [comment]
this LGTM

**Comment by flyingzumwalt** [comment]
Note: This overhaul also assumes that ipfs/community#153 will be done.

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: Thought you were going to rephrase it?

**Comment by flyingzumwalt** [comment]
I cleaned up the text a bit. This should be fine to use until the PM Process becomes more fully hammered out.

**Comment by flyingzumwalt** [comment]
@RichardLitt is it ok to just merge this now?

**Comment by RichardLitt** [comment]
Yes. The information isn't lost, so this is OK.

**Pull request merged**